lib/scripting/post-process/helpers.ts

```diff
@@ -17,304 +17,105 @@
-}
-
-export function indexOrParamsList1(result: [Expression[], Expression[]]): Expression[] {
-	const firstExpr = result[0];
-	const otherExprs = result[1];
-	return [...firstExpr, ...otherExprs];
-}
-
-export function indexOrParamsList2(result: [Expression[]]): Expression[] {
-	const expr = result[0];
-	return [...expr];
-}
-
-export function indexOrParams1(result: [Token, null, Expression, null, Expression[], Token]) {
-	const param = result[2];
-	const otherParams = result[4];
-	return [param, ...otherParams];
-}
-
-export function indexOrParams2(result: [Token, null, Expression[], Token]) {
-	const otherParams = result[2];
-	return [estree.literal(null), ...otherParams];
-}
-
-export function indexOrParams3(result: [Token, null, Expression, null, Token]) {
-	const param = result[2];
-	return [param];
-}
-
-export function indexOrParams4(result: [Token, null, Token]) {
-	return [];
-}
-
-export function indexOrParamsListDot1(result: [Expression[], Expression[]]): Expression[] {
-	const firstExpr = result[0];
-	const otherExprs = result[1];
-	return [...firstExpr, ...otherExprs];
-}
-
-export function indexOrParamsListDot2(result: [Expression[]]): Expression[] {
-	const expr = result[0];
-	return [...expr];
-}
-
-export function indexOrParamsDot1(result: [Token, null, Expression, null, Expression[], Token]) {
-	const param = result[2];
-	const otherParams = result[4];
-	return [param, ...otherParams];
-}
-
-export function indexOrParamsDot2(result: [Token, null, Expression[], Token]) {
-	const otherParams = result[2];
-	return [estree.literal(null), ...otherParams];
-}
-
-export function indexOrParamsDot3(result: [Token, null, Expression, null, Token]) {
-	const param = result[2];
-	return [param];
-}
-
-export function indexOrParamsDot4(result: [Token, null, Token]) {
+function ppArrayTypeModifiers(node: ESIToken): any {
 	return [];
 }
 
-export function commaExprList1(result: [Token, null, Expression, null, Expression[]]): Expression[] {
-	const firstExpr = result[2];
-	const otherExprs = result[4];
-	return [firstExpr, ...otherExprs];
-}
-
-export function commaExprList2(result: [Token, null, Expression[]]): Expression[] {
-	const exprs = result[2];
-	return [estree.literal(null), ...exprs];
-}
-
-export function commaExprList3(result: [Token, null, Expression, null]): Expression[] {
-	const expr = result[2];
-	return [expr];
-}
-
-export function commaExprList4(result: [Token, null]): Expression[] {
-	return [estree.literal(null)];
+function ppArraySizeInitializationModifier(node: ESIToken): any {
+	return node.children[1].estree;
 }
 
-export function nl(result: [EmptyStatement]): Comment[] {
-	const stmt = result[0];
-	return stmt.trailingComments ? stmt.trailingComments : [];
-}
-
-export function comment(result: [Token]): EmptyStatement {
-	let text = result[0].text;
-	text = text.substr(text.indexOf("'") + 1).trimRight();
-	return estree.emptyStatement([estree.comment('Line', text)]);
+function ppBoundList(node: ESIToken): any {
```

Comment:
just wondering - whats the pp prefix mean?

Reply:
To be honest it’s carry over from our nearley days and stood for post processor. I was going to get rid of the `pp` prefix, but I had some collisions with members of estree (example: ifStatement) so I figured I would just stay consistent